src/net.cpp

```diff
@@ -2276,6 +2276,10 @@ void ThreadMessageHandler()
 
         bool fSleep = true;
 
+        // randomly select node with whom to request mempoolsync
+        if (vNodesCopy.size() > 0)
+            requester.RequestMempoolSync(vNodesCopy[GetRandInt(vNodesCopy.size())]);
```

Comment:
Imo it would be better to only attempt to sync mempools with nodes that are not in IBD. nodes in IBD will under most circumstances have a mempool that is far less complete than one that has been "online" for a longer period of time and trying to sync with them will be a wasted effort.

Reply:
That's a good point. What's a good way to check if they're in IBD?